cmd/brew-test-bot.rb

```diff
@@ -1466,6 +1466,7 @@ def test_ci_upload(tap)
       if ARGV.include?("--dry-run")
         puts "git push --force #{remote} master:master :refs/tags/#{git_tag}"
       else
+        safe_system "git", "reset", "--hard", "origin/master"
         safe_system "git", "push", "--force", remote,
                                    "master:master", ":refs/tags/#{git_tag}"
```

Comment:
I have a simpler fix!
```suggestion
                                   "origin/master:master", ":refs/tags/#{git_tag}"
```
That one-line fix should do it. (and change the `--dry-run` code above)

Reply:
Yes, this seems like a nice idea 👍

Reply:
Thanks for the idea! :)